src/test/kotlin/minesweeper/domain/cell/CellTest.kt

```diff
@@ -5,12 +5,40 @@ import org.junit.jupiter.api.DisplayName
 import org.junit.jupiter.api.Test
 
 class CellTest {
+
+    private val defaultPosition = Pair(1, 1).toPosition()
+    private val otherPosition = Pair(2, 2).toPosition()
+
     @DisplayName(value = "Position이 같은 Cell은 동일한 Cell이다.")
     @Test
     fun checkSamePosition() {
-        val cell1 = Cell(Position(1, 1))
-        val cell2 = MineCell(Position(1, 1))
+        val cell1 = Cell(defaultPosition)
+        val cell2 = MineCell(defaultPosition)
         assertThat(cell1).isEqualTo(cell2)
         assertThat(cell1.hashCode()).isSameAs(cell2.hashCode())
```

Comment:
아래의 글을 읽고 코드에 반영하면 어떨까요?
https://meetup.toast.com/posts/185

```suggestion
        assertThat(cell1.hashCode()).isEqualTo(cell2.hashCode())
```